app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -10,106 +10,105 @@
-          app:layout_constraintStart_toEndOf="@+id/hint_title"
-          app:layout_constraintTop_toTopOf="parent"
-          app:layout_constraintVertical_bias="0.0"
-          android:visibility="@{viewModel.isHintRevealed? View.VISIBLE : View.GONE}">
+        <ImageView
+          android:id="@+id/hint_list_drop_down_icon"
+          android:layout_width="48dp"
+          android:layout_height="48dp"
+          android:layout_gravity="center_vertical"
+          android:contentDescription="@{@string/show_hide_hint_list(viewModel.hintsAndSolutionSummary)}"
+          android:padding="8dp"
+          android:src="@drawable/ic_arrow_drop_down_black_24dp"
+          app:isRotationAnimationClockwise="@{isListExpanded}"
+          app:rotationAnimationAngle="@{180f}" />
+      </FrameLayout>
+    </androidx.constraintlayout.widget.ConstraintLayout>
 
-          <ImageView
-            android:id="@+id/hint_list_drop_down_icon"
-            android:layout_width="48dp"
-            android:layout_height="48dp"
-            android:padding="8dp"
-            android:layout_gravity="center_vertical"
-            android:contentDescription="@{@string/show_hide_hint_list(viewModel.hintsAndSolutionSummary)}"
-            android:src="@drawable/ic_arrow_drop_down_black_24dp"
-            app:isRotationAnimationClockwise="@{isListExpanded}"
-            app:rotationAnimationAngle="@{180f}" />
-        </FrameLayout>
-      </androidx.constraintlayout.widget.ConstraintLayout>
+    <LinearLayout
+      android:id="@+id/hint_list_container"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="84dp"
+      android:layout_marginTop="8dp"
+      android:layout_marginEnd="84dp"
+      android:layout_marginBottom="32dp"
+      android:orientation="vertical"
+      android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
 
-      <LinearLayout
-        android:id="@+id/hint_list_container"
-        android:layout_width="match_parent"
+      <TextView
+        android:id="@+id/hints_and_solution_summary"
+        style="@style/Body"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="84dp"
-        android:layout_marginEnd="84dp"
-        android:layout_marginTop="8dp"
-        android:layout_marginBottom="32dp"
-        android:orientation="vertical"
-        android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
-
-        <TextView
-          android:id="@+id/hints_and_solution_summary"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_gravity="start"
-          android:fontFamily="sans-serif"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp" />
-      </LinearLayout>
+        android:layout_gravity="start"
+        android:fontFamily="sans-serif"
```

Comment:
You can remove `android:fontFamily="sans-serif"`. It's already getting implemented using the style.

Reply:
Done